profiling/triangle_strip.py

```diff
@@ -0,0 +1,41 @@
+from p5 import *
+
+x = 0
+y = 0
+outsideRadius = 150
+insideRadius = 100
+
+def setup():
+        size(720, 400)
+        background(204)
+
+        global x, y
+        x = width / 2
+        y = height / 2
+
+def draw():
+        global x, y, outsideRadius, insideRadius
+        background(204)
+
+        numPoints = 60
+        angle = 0
+        angleStep = 180.0 / numPoints
+
+        begin_shape("TRIANGLE_STRIP")
+
+        for i in range(numPoints):
```

Comment:
I think it should be `range(numPoints + 1)` as the strip is not being closed:

![Screenshot 2020-06-30 at 6 45 46 AM](https://user-images.githubusercontent.com/15258498/86071612-552f0c80-ba9d-11ea-90f9-7f5f3ec9ed3f.png)